python/hsfs/ge_validation_result.py

```diff
@@ -199,6 +202,13 @@ def validation_time(
             validation_time: The time at which validation was performed.
             Supported format include timestamps(int), datetime, date or string formatted to be datutils parsable.
         """
+        if isinstance(validation_time, str):
+            try:
+                validation_time = dateutil.parser.parse(validation_time).astimezone(
+                    datetime.timezone.utc
+                )
+            except ValueError:
+                pass
         if validation_time:
             self._validation_time = util.convert_event_time_to_timestamp(
                 validation_time
```

Comment:
I'm confused here. Why do we need this second if statement? Is there a case where the user sets the validation_time manually?

Reply:
I think for consistency with other places where we are dealing with start_time/end_time it is good to use the same function

Reply:
And removing it would loose compatibility if any user has been doing something where the time is set with something else than a str

Reply:
Ok fair enough. Maybe can you add a comment on why the second if is needed?